.gitignore

```diff
@@ -23,3 +23,5 @@ settings.yaml
 
 .idea
 
+# Desktop Service Store (macOS)
+.DS_Store
```

Comment:
Can we do a newline at the end of these ppc

Reply:
Updated!!

Reply:
I'll do it.